src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -98,7 +102,9 @@ public Table (String name, Class<? extends Entity> entityClass, Requirement requ
     }
 
     public static final Table AGENCY = new Table("agency", Agency.class, REQUIRED,
-        new StringField("agency_id",  OPTIONAL), // FIXME? only required if there are more than one
+        new StringField("agency_id",  OPTIONAL).requireConditions(
+            new ConditionalRequirement("agency_id", ROW_COUNT_GREATER_THAN_ONE)
```

Comment:
Whenever there is a conditional requirement like this, can you add a comment explaining what it means? Maybe even like to gtfs.org?

Reply:
Updated.